Blish HUD/Controls/Control.cs

```diff
@@ -793,8 +786,6 @@ public static class StandardColors {
                     this.DisposeControl();
                 }
 
-                // TODO: free unmanaged resources (unmanaged objects) and override a finalizer below.
-                // TODO: set large fields to null.
                 _disposedValue = true;
```

Comment:
can you give a quick comment why this is not needed anymore?

Reply:
It was a misunderstanding of mine at the time that it would assist with GC to clear some of the larger fields, but this of course is done automatically.

The reference to freeing unmanaged resources is only relevant to the TextBox control (and soon won't be with the new work being done on the TextBox control) and the CachedRender stuff.  Both of these already handle freeing unmanaged resources when disposed as described.